src/index.html

```diff
@@ -2,11 +2,383 @@
+              </div>
+              <span class="card__rating-reviews">Reviews: 5</span>
+            </div>
+    
+            <div class="card__price">
+              <span class="card__price-name">Price:</span>
+              <span class="card__price-value">$2,199</span>
+            </div>
+            <a 
+              href="/"
+              data-qa="card-hover"
+            >
+              <button class="card__button">Buy</button>
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img 
+            class="card__photo" 
+            src="./images/imac.jpeg" 
+            alt="APPLE A1419 iMac/27"
+            width="160"
+            height="134"
+          >
+        
+          <div class="card__info">
+            <h2 class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <span class="card__rating-reviews">Reviews: 5</span>
+            </div>
+    
+            <div class="card__price">
+              <span class="card__price-name">Price:</span>
+              <span class="card__price-value">$2,199</span>
+            </div>
+            <a 
+              href="/"
+              
+            >
+              <button class="card__button">Buy</button>
+            </a>
+          </div>
+        </div>
+
+
+        <div class="card">
+          <img 
+            class="card__photo" 
+            src="./images/imac.jpeg" 
+            alt="APPLE A1419 iMac/27"
+            width="160"
+            height="134"
+          >
+        
+          <div class="card__info">
+            <h2 class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <span class="card__rating-reviews">Reviews: 5</span>
+            </div>
+    
+            <div class="card__price">
+              <span class="card__price-name">Price:</span>
+              <span class="card__price-value">$2,199</span>
+            </div>
+            <a 
+              href="/"
+              
+            >
+              <button class="card__button">Buy</button>
+            </a>
```

Comment:
do you really need a link and button inside? it would be better to leave just the link and set it `card__button` class
```suggestion
            <a href="/" class="card__button">
              Buy
            </a>
```

Reply:
and please check all others places